setup.py

```diff
@@ -49,7 +49,7 @@ def is_requirement(line):
 
 setup(
     name='lti-consumer-xblock',
-    version='2.9.1',
+    version='2.9.2',
```

Comment:
Please add an entry to the history in the readme.

Reply:
Done!